directgov_passportapplication.yml

```diff
@@ -0,0 +1,8 @@
+---
+site: directgov_passportapplication
+whitehall_slug: hm-passport-office
+homepage: https://passportapplication.service.gov.uk
+tna_timestamp: 20201010101010 # Stub timestamp - site not in TNA
+host: passportapplication.direct.gov.uk
+global: =301 https://passportapplication.service.gov.uk
+global_redirect_append_path: true
```

Comment:
Thanks, @jennyd - you're right. We don't need to append the path. I've removed that.